less/transferwise.less

```diff
@@ -142,3 +141,27 @@ li .circle {
     color: @brand-smoke-minus-20;
   }
 }
+
+/**
+ * Notification center
+ */
+
+@media screen and (max-width: 767px) {
```

Comment:
We also need to use the LESS variables for the width in case we change these values in future.  They're something like @screen-sm-max - take a look in the variables.less file.